IdentityCore/src/webview/embeddedWebview/ui/mac/MSIDWebviewUIController.m

```diff
@@ -44,14 +44,26 @@ + (void)initialize
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
         s_webConfig = [WKWebViewConfiguration new];
-        
+        s_webConfig.applicationNameForUserAgent = kMSIDPKeyAuthKeyWordForUserAgent;
         if (@available(macOS 10.15, *))
         {
             s_webConfig.defaultWebpagePreferences.preferredContentMode = WKContentModeDesktop;
         }
     });
 }
 
++ (WKWebViewConfiguration *)createWebViewConfigWithPKeyAuthUserAgent
```

Comment:
can initialize method reuse this API to avoid duplicate code?

Reply:
Good idea.

Reply:
Curious to know, if `defaultWKWebviewConfiguration` is only being called from the singleton in `initialize`, can the method code be shifted to the singleton. Any other class calling this method won't really give the config for this controller.

Reply:
@ameyapat 
Thank you for your comment. 
If we make it private, it won't be accessible in the public ADAL API. I think the usage case would be that OneDrive team could initialize the webview instance and then add the config setting they get from this API using 

WKWebViewConfiguration *webConfig = [MSIDWebviewUIController defaultWKWebviewConfiguration];

WKWebView *webView = [[WKWebView alloc] initWithFrame:rootView.frame configuration:webConfig];